modules/oe_webtools_analytics/src/Form/WebtoolsAnalyticsSettingsForm.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\oe_webtools_analytics\Form;
+
+use Drupal\Core\Form\ConfigFormBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Settings form for module.
+ */
+class WebtoolsAnalyticsSettingsForm extends ConfigFormBase {
+
+  /**
+   * Name of the config being edited.
+   */
+  const CONFIG_NAME = 'oe_webtools_analytics.settings';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function getFormId() {
+    return 'oe_webtools_analytics_settings';
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildForm(array $form, FormStateInterface $form_state) {
+    $form['siteID'] = [
+      '#type' => 'number',
+      '#title' => $this->t('Site ID'),
+      '#default_value' => $this->config(static::CONFIG_NAME)->get('siteID'),
```

Comment:
Instead of `static::CONFIG_NAME` let's use `WebtoolsAnalyticsSettingsForm::CONFIG_NAME`.